FWCore/ParameterSet/interface/validateTopLevelParameterSets.h

```diff
@@ -0,0 +1,17 @@
+#ifndef FWCore_ParameterSet_validateTopLevelParameterSets_h
+#define FWCore_ParameterSet_validateTopLevelParameterSets_h
+
+namespace edm {
+
+  class ParameterSet;
+  class ParameterSetDescription;
+
+  const unsigned int s_defaultNumberOfThreads = 1;
```

Comment:
How about `constexpr`

Reply:
Done. I pushed a change that makes it constexpr.